Alpha_shapes_2/doc/Alpha_shapes_2/Alpha_shapes_2.txt

```diff
@@ -43,6 +43,10 @@ it's way too large. So we will never spoon up ice-cream lying in the
 inside of the convex hull of \f$ S\f$, and hence the \f$ \alpha\f$-shape for
 \f$ \alpha \rightarrow \infty\f$ is the convex hull of \f$ S\f$.
 
+\cgal offers 2D and 3D alpha shapes. The GUDHI library offers a
+<a href="http://gudhi.gforge.inria.fr/doc/latest/group__alpha__complex.html"> dD Alpha complex</a>.
```

Comment:
As far as I know, CGAL has no persistence computation module, neither data structure adapted to such comptutations. Am I wrong ?
GUDHI's team is grateful of all the work done on CGAL, and also to contribute to CGAL development.